packages/params/src/presets/minimal/index.ts

```diff
@@ -2,10 +2,12 @@ import {IBeaconPreset} from "../../preset";
 
 import {phase0} from "./phase0";
 import {altair} from "./altair";
+import {merge} from "./merge";
 
-export const commit = "v1.1.0-alpha.7";
+export const commit = "v1.1.4";
```

Comment:
What is this variable used for?

Reply:
this commit is used to pull the preset files from the spec and compare the values in the params test:
https://github.com/ChainSafe/lodestar/blob/08dbb21538bea1f13aa64bdcebe76ec4bfaa4249/packages/params/test/e2e/ensure-config-is-synced.test.ts#L10